### DISCUSSION TITLE
Add Account Verification Similar to osu!stable

### DISCUSSION THREAD

**Discussion by itsskyballs** [ppy/osu#25372] [Ideas] [answered]
As a dedicated Osu! Lazer player, I appreciate the game's commitment to security and integrity. In line with this, I'd like to propose the implementation of account verification similar to Osu!stable.

I kindly request the addition of account verification mechanisms in Osu! Lazer that mirror the security features present in Osu!stable. This would include email verification, two-factor authentication (2FA), or any other measures that enhance the security of player accounts.

**Why This Feature Would Be Valuable:**
1. Account verification adds an extra layer of protection against unauthorized access to player accounts.
2. It can help prevent fraudulent activities, such as hacking or account theft.
3. Account security is a paramount concern for players, and implementing these measures would instill confidence in the community.

By introducing account verification features similar to Osu!stable, Osu! Lazer can maintain a high standard of account security, reassuring players that their accounts and progress are safe.

Thank you for considering this request. The addition of account verification would be a valuable enhancement to Osu! Lazer's security measures.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/20590
